api/envoy/config/filter/network/tcp_proxy/v2/README.md

```diff
@@ -0,0 +1,28 @@
+Protocol buffer definitions for the TCP Proxy.
```

Comment:
Is there any reason to not just do this inline in the docs in the filter configuration section? We have other samples in there and an MD seems a bit out of place?